enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -21,174 +21,124 @@ see-also:
 
 #### As a service provider, I want to forward selected API audit logs to my chosen destinations
+
 - configure the SD internal audit policy
 - configure API audit policy and forwarding using name-spaced logging customer resources in the management cluster.
-  - for example: Red Hat Service Delivery team forwards audit logs to internal splunk destination.
+  - for example: Red Hat Service Delivery team forwards audit logs to internal Splunk destination.
 - independent of user configuration.
 
 ### Goals
 
 - Multiple audit policies can be defined in a management cluster, independent of hosted cluster policies.
 - Multiple audit policies can be defined in a hosted cluster, independent of the management cluster policies.
   See [api-audit-log-policy](/api-audit-log-policy.md) for how the CLO will support this.
-- Allow some forwarding in the _management_ cluster to be configured from the _hosted_ cluster.
-- Keep direct control of management cluster forwarding on the management cluster.
-  - hosted cluster has no direct access to control plane configuration.
-  - data plane resources can request forwarding by the management cluster.
-  - management cluster HLO automatically validates data-plane requests, and enables them if they are safe.
+- Data plane (customer) requests audit log forwarding by modifying data-plane CLF configuration, forwarding from control plane starts automatically.
 
 ### Non-Goals
 
 - This proposal only covers API audit logs. No change to application, infrastructure or node audit logs.
-- Management cluster is not _required_ to honour all possible forwarding configurations allowed by CLF. \
 
 ## Proposal
 
 ### Workflow Description
 
 #### Hosted Cluster, data plane
 
-1. Create a `Policy.audit.k8s.io` resource with the desired policy
-   (see [api-audit-log-policy](/api-audit-log-policy.md))
-1. Create a `HypershiftLogForwarder` (HLF) resource with an `audit` input linked to the policy.
-1. Create HLF `outputs` and `pipelines` to forward the resulting logs.
-1. Optionally create a `ClusterLogForwarder` to forward other logs (not API-audit) as usual.
+1. Install the Cluster Logging Operator (CLO).
+1. Create a `ClusterLogForwarder` with a HTTP input.
+ - Receiver should be identified with a special input name e.g. `hypershiftAPIAudit`
+ - Pipelines from the receiver may include an [api-audit-log-policy](/api-audit-log-policy.md) filter.
+1. Forward logs to any outputs as desired.
 
 #### Management Cluster, control plane
 
-On the _management_ cluster
-1. Install the Cluster Logging Operator (CLO), watching all name-spaces.
-1. Install the new Hypershift Logging Operator (HLO), watching all name-spaces.
-1. In each new HCP name-space, create a `ClusterLogForwarderTemplate` resource. \
-   This resource may be identical for all HCPs or customized per HCP.
+1. Install the Cluster Logging Operator (CLO).
+1. Create a `ClusterLogForwarder` for each HCP to forward to desired outputs.
```

Comment:
I remember at the beginning, we have limitation that there can be only one CLF to be consumed by CLO.
So with the latest design, there will be no such limitation, and each CLF can define the separated forwording rule and work together?

Reply:
+1 @bmeng this helps us avoid the CLFT or similar on the management side. Instead of a more complex template that includes splunk + http forwarding, we can separate distinct forwarding tasks into separate CLF configurations. We don't _have_ to separate them, but it makes the individual configurations simpler and provides better isolation between the tasks in case a forwarder fails.